src/program/validation/checkers/token_checkers.rs

```diff
@@ -97,6 +97,15 @@ impl<'a, 'info> TokenAccountInfo<'a, 'info> {
     }
 }
 
+impl<'a, 'info> TokenAccountInfo<'a, 'info> {
+    pub fn amount(&self) -> Result<u64, ProgramError> {
```

Comment:
is this copied from somewhere?

if so, source would be nice
if not, struct def in here would be nice

Reply:
read PR description: copied from Anchor